src/ap/wpa_auth_kay.c

```diff
@@ -325,7 +325,7 @@ int ieee802_1x_alloc_kay_sm_hapd(struct hostapd_data *hapd,
 	kay_ctx->enable_transmit_sa = hapd_enable_transmit_sa;
 	kay_ctx->disable_transmit_sa = hapd_disable_transmit_sa;
 
-	res = ieee802_1x_kay_init(kay_ctx, policy,
+	res = ieee802_1x_kay_init(kay_ctx, policy, 0, 0, 1,
```

Comment:
wonder if we can use macro here?

Reply:
This portion of code is part of hostapd, not wpa-supplicant.  This change was needed for compilation of wpa debian package which builds both hostapd and wpa-supplicant, even though only wpa-supplicant was installed for SONiC test.  Since sonic-wpa-supplicant is not currently incorporated in sonic-buildimage, it is not clear if hostapd will be included in eventual SONiC build/install and this change will still needed.